.github/workflows/release.yaml

```diff
@@ -0,0 +1,51 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+env:
+  go-version: 1.16
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Setup golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+
+      - name: Build binaries
+        run: make install GOBIN=`pwd`/docker
+
+      - name: Build setup-hw image
+        run: |
+          docker build -t quay.io/cybozu/setup-hw:latest --target stage1 docker
+          docker build -t quay.io/cybozu/setup-hw-secret:latest docker
+
+      - name: Push images
+        run: |
+          echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+          TAG=${GITHUB_REF#refs/tags/v}
+          for name in setup-hw setup-hw-secret; do
+              echo "pushing ${name}:v${TAG} ..."
+              docker tag quay.io/cybozu/${name}:latest quay.io/cybozu/${name}:$TAG
+              docker push quay.io/cybozu/${name}:latest
```

Comment:
I don't like `latest` tag in general.

Reply:
I am not sure it is good to remove `latest` tag altogether.
If we remove `latest` tag, we need to update README.md in every upgrades and that might be annoying.

Reply:
fair enough.

Reply:
@binoue 
However, I don't accept to update `latest` with RC images.